pkg/minikube/tunnel/registry_test.go

```diff
@@ -251,6 +251,12 @@ func tmpFile(t *testing.T) string {
 	if err != nil {
 		t.Errorf("failed to create temp file %s", err)
 	}
+	defer func() { //clean up tempdir
```

Comment:
this is wrong, because the temp file is deleted when the tempFile func is finished, and the called wont have the chance to this this temp file

Reply:
If you are correct, also each function already deletes those files, it is not necessary to use that `defer`